scripts/linters/pre_commit_linter.py

```diff
@@ -196,20 +196,13 @@ def _get_data(self, filepath, mode):
         return self._CACHE_DATA_DICT[key]
 
 
-def _lint_all_files(
-        js_filepaths, ts_filepaths, py_filepaths, html_filepaths,
-        css_filepaths, file_extensions_to_lint, verbose_mode_enabled=False):
+def _lint_all_files(file_extensions_to_lint, verbose_mode_enabled=False):
     """Run all lint checks.
 
     Args:
-        js_filepaths: list(str). The list of js filepaths to be linted.
-        ts_filepaths: list(str). The list of ts filepaths to be linted.
-        py_filepaths: list(str). The list of python filepaths to be linted.
-        html_filepaths: list(str). The list of HTML filepaths to be linted.
-        css_filepaths: list(str). The list of CSS filepaths to be linted.
-        verbose_mode_enabled: bool. True if verbose mode is enabled.
         file_extensions_to_lint: list(str). The list of file extensions to be
             linted.
+        verbose_mode_enabled: bool. True if verbose mode is enabled.
 
     Returns:
         custom_linter: list. Custom lint checks.
```

Comment:
for lines 212 and 214: execute it when required.

Reply:
Done.